python/activity_stream/activity_stream.py

```diff
@@ -412,6 +496,39 @@ def load_data(self, sg_entity_dict):
         self._bundle.log_debug("Ask db manager to ask shotgun for updates...")
         self._data_manager.rescan()
         self._bundle.log_debug("...done")
+
+    def show_new_note_dialog(self, modal=True):
+        """
+        Shows a dialog that allows the user to input a new note.
+
+        :param modal:   Whether the dialog should be shown modally or not.
+        :type modal:    bool
+        """
+        note_dialog = NoteInputDialog(parent=self)
+        note_dialog.entity_created.connect(self._on_entity_created)
+        note_dialog.data_updated.connect(self.rescan)
+        note_dialog.set_bg_task_manager(self._task_manager)
+        note_dialog.set_current_entity(self._entity_type, self._entity_id)
+
+        if modal:
+            note_dialog.exec_()
```

Comment:
After another, look, I don't think we should worry about it here. The activity stream widget emits an entity_created signal, which is what callers should be listening for rather than relying on the return value of the dialog. I'll be noting that in the docs.